github/orgs_teams.go

```diff
@@ -114,10 +114,37 @@ func (s *OrganizationsService) GetTeam(ctx context.Context, team int) (*Team, *R
 	return t, resp, nil
 }
 
+// NewTeam represents a team to be created or modified.
+type NewTeam struct {
+	Name         string   `json:"name"` // Name of the team (Required.)
```

Comment:
Nitpicky, but I'd add a period at the end of the sentence. "Required" is another sentence inside parenthesis.

```Go
// Name of the team. (Required.)
```

Reply:
This how it's written elsewhere. See:

https://github.com/google/go-github/blob/511f540f1887d30b88cee4a2fcd1f2922754acf4/github/repos.go#L552